### PR TITLE
Bot Thoughts

### PR DESCRIPTION
**Pull request by Penguin2600**
Adds Bot thoughts command
Adds Requirements file for requirements tracking

### CODE REVIEW

**Comment by DavidCaudill** [comment]
A fully working feature in just one commit? +1

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+hangups
+requests
```
Comment (PHLAK) [inline requirements.txt:2]: Should just add these to the README.

**Comment by Penguin2600** [comment]
Tested even.

File: requirements.txt (hunk 1 above)
Comment (Penguin2600) [inline requirements.txt:2]: I disagree.
https://pip.readthedocs.org/en/1.1/requirements.html

File: hangupsbot/commands.py [hunk 2]
```diff
@@ -304,3 +305,36 @@ def roll(bot, event, *args):
             i += 1
         message = 'Rolling {}: {} ({})'.format(arg, results, total)
         bot.parse_and_send_segments(event.conv, message)
+
+
+def get_topic(seed):
```
Comment (PHLAK) [inline hangupsbot/commands.py:310]: Why do all this outside of the 'thoughts' function below?

Comment (Penguin2600) [inline hangupsbot/commands.py:310]: Cleaner. Really we should adjust all the commands like this to return strings like I have done and then make one "bot send" function, would be cleaner.

**Comment by PHLAK** [comment]
Overall good, shipit! :+1:

File: requirements.txt (hunk 1 above)
Comment (DavidCaudill) [inline requirements.txt:2]: that's not how a requirements.txt works. Pip3 install -r requirements.txt

Comment (PHLAK) [inline requirements.txt:2]: TIL

File: hangupsbot/commands.py (hunk 2 above)
Comment (PHLAK) [inline hangupsbot/commands.py:310]: Would also be nice to somehow build each command in a separate file that is automatically included somehow.  The commands.py file will get large and disorganized fast.

**Pull request merged**